src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.ps1

```diff
@@ -0,0 +1,15 @@
+ param (
+    [Parameter(Mandatory=$true)][string]$ToolRuntimePath,
+    [Parameter(Mandatory=$true)][string]$DotnetCmd
+ )
+
+# Override versions in runtimeconfig.json files with highest available runtime version.
+$mncaFolder = (gi $DotnetCmd).Directory.FullName + "\shared\Microsoft.NETCore.App"
```

Comment:
\<nit> `gi`: I usually see style guides saying "don't use aliases in scripts" for powershell stuff, but I don't know if that's generally followed.